core-ui/src/components/Lambdas/LambdaDetails/Tabs/Configuration/ServiceBindings/CreateServiceBindingForm.js

```diff
@@ -41,7 +41,7 @@ export default function CreateServiceBindingForm({
   }, [setValidity]);
 
   useEffect(() => {
-    if (!existingInstanceName || !serviceBindings?.length) {
+    if (!existingInstanceName) {
```

Comment:
Why have you removed it? If you try to create binding on brand new namespace (without any service bindings the view will crash)

Reply:
something went wrong with merging from `main`. I'll revert this change